### PR TITLE
Prep for Release 3.0.0

### PR DESCRIPTION
**Pull request by EdwinGuzman**
Fixes JIRA ticket xxx

## This PR does the following:

- Prep for Release 3.0.0

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -6,28 +6,31 @@ The format is based on [Keep a Changelog](https://keepachangelog.com/en/1.0.0/).
 Currently, this repo is in Prerelease. When it is released, this project will adhere to [Semantic Versioning](https://semver.org/spec/v2.0.0.html)
 ========
 
-## React 18 / Chakra 2.8 Release
+## 3.0.0 (March 14, 2024)
+
+### React 18 / Chakra 2.8 Release
```
Comment (bigfishdesign13) [inline CHANGELOG.md]: Maybe combine these?
```
## 3.0.0 (March 14, 2024) - React 18 / Chakra 2.8
```

File: src/components/Accordion/accordionChangelogData.ts [hunk 2]
```diff
@@ -10,8 +10,8 @@ import { ChangelogData } from "../../utils/ComponentChangelogTable";
 
 export const changelogData: ChangelogData[] = [
   {
-    date: "Prerelease",
-    version: "Prerelease",
+    date: "2014-03-14",
```
Comment (bigfishdesign13) [inline src/components/Accordion/accordionChangelogData.ts]: All of the dates are using `2014`.

**Review by bigfishdesign13 — CHANGES_REQUESTED** [review]
There was a typo in your find-and-replace string for the date.

I had seen that the changelog updates were starting to overlap in weird ways, but it looks like you updated things accordingly.

File: src/components/Accordion/accordionChangelogData.ts (hunk 2 above)
Comment (EdwinGuzman) [reply src/components/Accordion/accordionChangelogData.ts]: WHAT! ugh ok

Comment (EdwinGuzman) [reply src/components/Accordion/accordionChangelogData.ts]: Updated.

**Review by bigfishdesign13 — APPROVED** [review]
Release the hounds!

**Pull request merged**